src/model/control_desk.py

```diff
@@ -284,28 +311,32 @@ def unlink(self) -> bool:
             else:
                 i += 1
         BankSet._fish_connector.send_fader_bank_set_delete_message(self.id)
-        BankSet._linked_bank_sets.pop(found_index)
+        BankSet._linked_bank_sets.pop(found_index)  # TODO found index koente leer sein
         return True
 
     @property
     def is_linked(self) -> bool:
+        """TODO docstring"""
         return self.pushed_to_fish
 
 
 def set_network_manager(network_manager: NetworkManager):
+    """TODO docstring"""
```

Comment:
"""Set the network manager instance to be used by all bank sets and subsequent banks and columns."""